geom/src/duration.rs

```diff
@@ -81,7 +85,7 @@ impl Duration {
         remainder -= minutes * 60.0;
         let seconds = remainder.floor();
         remainder -= seconds;
-        let centis = (remainder / 0.1).floor();
+        let centis = (remainder / 0.01).round();
```

Comment:
I think this was a bug - centi should be `seconds * 100` right?

Reply:
Eep -- how did this get by for so long? Maybe I meant "deciseconds". Looking at the call sites, winding up with centiseconds looks fine.

Reply:
I was wrong -- we do only want to print one decimal place. Loading screens with two decimal places are kind of overwhelming. :)   Fixed in 7399f2aff2ed0f61f8202bfadc721df5078e7583